src/contributors/judithparedes.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Judith Mishel Paredes Artica", 
+    "desc": "Front End Developer",
+    "github" : "https://github.com/mishelpa",
```

```suggestion
    "github": "https://github.com/mishelpa",
```
Remove unwanted white-space.